sihl/src/database_migration.ml

```diff
@@ -220,10 +221,68 @@ struct
     execute steps
   ;;
 
+  let migrations_status () =
+    let open Lwt.Syntax in
+    let* migrations_states = Repo.get_all (table ()) in
+    let migration_states_namespaces =
+      migrations_states
+      |> List.map (fun migration_state ->
+             migration_state.Database_migration_repo.Migration.namespace)
+    in
+    let registered_migrations_namespaces =
+      Map.to_seq !registered_migrations |> List.of_seq |> List.map fst
+    in
+    let namespaces_to_check =
+      List.concat
+        [ migration_states_namespaces; registered_migrations_namespaces ]
```

Comment:
You have some duplicate namespaces in this list, right? Maybe you could get rid of them

Reply:
Nice one, fixed.